### ISSUE TITLE
Set SQL Server connection for all users by default

### ISSUE THREAD

**Issue by deanoblock** [mRemoteNG/mRemoteNG#2498]
Use case is where I have many users remoting into a Server,  set mRemoteNG to use the SQL Server connection settings by default.  Where I can maintain a list of machines on the shopfloor and support users can login and load the SQL Server connections automatically without having to manually configure the advanced settings.

## Feature request Behavior
1.New user login to a windows server for the first time
2. Windows User Profile is created
3. User opens mRemoteNG
4. SQL Server connection is used by default

## Current Behavior
1.New user login to a windows server for the first time
2. Windows User Profile is created
3. User opens mRemoteNG
4. User is prompted to open or start a new local XML connection file

## Possible Solution
Allow SQL Server connection setting to be global and not user based

## Context
I have many users logging in to a server (new contractors) and I want them to all open mremoteng and connect to the SQL Server Db by default without having to manually add the SQL server connection info.  Because they are low level users that do not know what a SQL Connection is.

## Your Environment
Manufacturing shopfloor, robotic equipment

**Comment by simonai1254** [comment]
How do you deploy the software onto the user's clients? 
If I remember correctly (need to check), mRemoteNG has two configuration files. One is the user configuration, and the other is the connection config. Both files are stored in the Appdata folder of mRemoteNG. If you create a default user configuration file containing the Database settings, you can just replace the file of the user and he's ready to go.

**Comment by deanoblock** [comment]
This is on Microsoft Server 2022 with remote desktop services.  Since mRemoteNG is per machine installation, any user new or existing logging in has the application already.   
I think I know where you are going with this, I can create a default configs then create a local logon batch script to copy them into the users %appdata% location.  A little messy but I'll give it try.  However I still think its a useful FR to have the connection config live in $programdata

**Comment by simonai1254** [comment]
Got it. In this case the only option (workaround until this feature is added) I see is to alter the default profile, but this obviously only works if the profile was not created before and roaming profile is disabled...

**Comment by Kvarkas** [comment]
with profiles added it should be posible, will keep this in mind on feature developing.

**Comment by VQFCOSTA-Infortech** [comment]
@deanoblock 

Can you share the database schema and the mremoteng version you use?

I'm having errors when using the MySQL I use because of the schema.

#2500

**Comment by fpaulet** [comment]
I've installed the tool for my support team. The tool is using a SQL Server DB; connection with a global user (SQL Server user) and all is working fine.
I'm trying to connect to the DB with a Windows user but I'm receiving a connection error.
Do you know if that functionality is already implemented?
Thanks

**Comment by Kvarkas** [comment]
> I've installed the tool for my support team. The tool is using a SQL Server DB; connection with a global user (SQL Server user) and all is working fine. I'm trying to connect to the DB with a Windows user but I'm receiving a connection error. Do you know if that functionality is already implemented? Thanks

if you try to do connection to same db but from cmd with that user is it successfully? I expect there could be limitation on db side for such, need to check

**Comment by fpaulet** [comment]
Yes, the user is working well on the DB. But we are talking about 'Windows user' and not 'SQL Server' user. I do not have any issue using a SQL Server user. But I would like to let the users use their Windows user.
Thanks
